tests/test_neurips_conference.py

```diff
@@ -1020,6 +1024,74 @@ def test_setup_matching(self, conference, client, helpers):
         # print(url)
         # assert False
 
+    def test_ac_reassignment(self, conference, helpers, client):
+
+        pc_client=openreview.Client(username='pc@neurips.cc', password='1234')
+        submissions=conference.get_submissions()
+
+        assert len(pc_client.get_edges(invitation='NeurIPS.cc/2021/Conference/Senior_Area_Chairs/-/Assignment')) == 3
+        assert len(pc_client.get_edges(invitation='NeurIPS.cc/2021/Conference/Area_Chairs/-/Assignment')) == 5
+
+        ac_assignment = pc_client.get_edges(invitation='NeurIPS.cc/2021/Conference/Area_Chairs/-/Assignment', head=submissions[0].id)[0]
+
+        ## Remove AC assignment
+        ac_assignment.ddate = openreview.tools.datetime_millis(datetime.datetime.utcnow())
+        pc_client.post_edge(ac_assignment)
+
+        helpers.await_queue()
+
+        assert len(pc_client.get_edges(invitation='NeurIPS.cc/2021/Conference/Senior_Area_Chairs/-/Assignment')) == 3
+        assert len(pc_client.get_edges(invitation='NeurIPS.cc/2021/Conference/Area_Chairs/-/Assignment')) == 4
+
+        assert [] == pc_client.get_group('NeurIPS.cc/2021/Conference/Paper5/Area_Chairs').members
+        assert [] == pc_client.get_group('NeurIPS.cc/2021/Conference/Paper5/Senior_Area_Chairs').members
+
+        ## Add AC assignment
+        ac_assignment.ddate = None
+        pc_client.post_edge(ac_assignment)
+
+        helpers.await_queue()
+
+        assert len(pc_client.get_edges(invitation='NeurIPS.cc/2021/Conference/Senior_Area_Chairs/-/Assignment')) == 3
+        assert len(pc_client.get_edges(invitation='NeurIPS.cc/2021/Conference/Area_Chairs/-/Assignment')) == 5
+
+        assert ['~Area_IBMChair1'] == pc_client.get_group('NeurIPS.cc/2021/Conference/Paper5/Area_Chairs').members
+        assert ['~SeniorArea_GoogleChair1'] == pc_client.get_group('NeurIPS.cc/2021/Conference/Paper5/Senior_Area_Chairs').members
+
+        sac_assignment = pc_client.get_edges(invitation='NeurIPS.cc/2021/Conference/Senior_Area_Chairs/-/Assignment', head='~Area_IBMChair1')[0]
+        assert sac_assignment.tail == '~SeniorArea_GoogleChair1'
+
+        ## Add SAC assignment
+        sac_assignment.ddate = openreview.tools.datetime_millis(datetime.datetime.utcnow())
+        pc_client.post_edge(sac_assignment)
+
+        helpers.await_queue()
+
+        assert len(pc_client.get_edges(invitation='NeurIPS.cc/2021/Conference/Senior_Area_Chairs/-/Assignment')) == 2
+        assert len(pc_client.get_edges(invitation='NeurIPS.cc/2021/Conference/Area_Chairs/-/Assignment')) == 5
```

Comment:
If you remove an SAC, the AC assignment does not change?

Reply:
the AC shouldn't never change by an SAC change. This test removes the SAC assignment, meaning the AC doesn't have an SAC anymore, that's why the SAC paper groups are empty. The PCs must find another SAC for that AC and then the group will have the new SAC as member.

Reply:
Yes, OK the same explanation as before the SAC is linked to the submission through the AC. Thanks!